administration/src/cards/CardBlueprint.ts

```diff
@@ -102,9 +104,7 @@ export class CardBlueprint implements JSONCardBlueprint {
     return new CardInfo({
       fullName: this.fullName,
       expirationDay:
-        expirationDate !== null && expirationDate.getTime() > 0 && !this.hasInfiniteLifetime()
```

Comment:
`expirationDate.getTime()` this check is required. If you type in the dateInput field and start typing the year it could be `0002` for the first digit which leads to negative time and protobuf fails
`invalid uint 32: -24681`